### PR TITLE
Python 3.7 support

### PR DESCRIPTION
**Pull request by ololobus**
Python 3.7 support https://github.com/deepmipt/DeepPavlov/issues/520

Scikit-learn version changed to 0.19.2 in order to address this issue https://github.com/scikit-learn/scikit-learn/issues/11320

pyTelegramBotAPI version changed to the latest to fix test errors, e.g.:
```python
E             File "/Users/alexk/dev/deeppavlov-dev/utils/telegram_utils/telegram_ui.py", line 18, in <module>
E               import telebot
E             File "/Users/alexk/dev/deeppavlov-dev/dp-venv/lib/python3.7/site-packages/telebot/__init__.py", line 1291
E               @util.async()
E                         ^
E           SyntaxError: invalid syntax
```

Pip install and basic functionality works fine with Python 3.7.0 now. Tests do not crash, but take forever on my laptop to complete and some of them are skipped, though the same is with current master with Python 3.6.6.

### CODE REVIEW

File: tests/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+pytest>=3.8.2
+pexpect>=4.6.0
```
Comment (yoptar) [inline tests/requirements.txt]: Tests requirements are set as extra dependencies in the `setup.py` file:
https://github.com/deepmipt/DeepPavlov/blob/5b2132709ce51c370629a66b2296033dc5e73d99/setup.py#L57-L60

**Comment by yoptar** [comment]
Thank you for your contribution!
But i thought that TensorFlow does not support Python 3.7 as of yet, so most of our models would still not work with it.

**Comment by ololobus** [comment]
Ok, TF is not directly listed in the requirements.txt, so I thought it is not crytical

**Comment by aranyasinha** [comment]
Tensorflow has supported python3.7 for some time now.

**Comment by nok** [comment]
@aranyasinha is right, now it's possible: https://www.tensorflow.org/install/pip?lang=python3#package-location

**Comment by yoptar** [comment]
There is a PR #861, it is planned to get merged and released this week.

**Pull request closed without merging**